cmake/onnxruntime_graph.cmake

```diff
@@ -32,18 +32,47 @@ if(NOT onnxruntime_USE_DML)
 endif()
 
 file(GLOB_RECURSE onnxruntime_ir_defs_src CONFIGURE_DEPENDS
-    "${ONNXRUNTIME_ROOT}/core/defs/*.cc"
+  "${ONNXRUNTIME_ROOT}/core/defs/*.cc"
 )
 
-add_library(onnxruntime_graph ${onnxruntime_graph_src} ${onnxruntime_ir_defs_src})
+if (onnxruntime_ENABLE_TRAINING)
+  file(GLOB_RECURSE orttraining_graph_src CONFIGURE_DEPENDS
+      "${ORTTRAINING_SOURCE_DIR}/core/graph/*.h"
```

Comment:
Is it possible to split orttraining_graph_src into a separated static lib?

Reply:
possibly, what's the benefit?

---
In reply to: [392566041](https://github.com/microsoft/onnxruntime/pull/3174#discussion_r392566041) [](ancestors = 392566041)